dataservice/api/biospecimen/models.py

```diff
@@ -0,0 +1,78 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base, KfId
+from dataservice.api.genomic_file.models import GenomicFile
+
+
+class Biospecimen(db.Model, Base):
+    """
+    Biospecimen entity.
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param external_sample_id: Name given to sample by contributor
+    :param external_aliquot_id: Name given to aliquot by contributor
+    :param composition : The cellular composition of the biospecimen.
+    :param tissue_type: description of the kind of tissue collected
+           with respect to disease status or proximity to tumor tissue
+    :param anatomical_site : The name of the primary disease site of the
+           submitted tumor biospecimen
+    :param age_at_event_days: Age at the time biospecimen was
+            acquired, expressed in number of days since birth
+    :param tumor_descriptor: The kind of disease present in the tumor
+           specimen as related to a specific timepoint
+    :param shipment_origin : The origin of the shipment
+    :param shipment_destination: The destination of the shipment
+    :param analyte_type: Text term that represents the kind of molecular
+           specimen analyte
+    :param concentration: The concentration of an analyte or aliquot extracted
```

Comment:
What did we decide in terms of standard units for concentration and volume? From the X01 data so far I see that concentration is usually in micrograms per microliter and volume is in microliters. 
@allisonheath

Reply:
I'm actually wondering if we should add unit fields here? For volume it can be different based on whether it's liquid (blood - typically microliters) or solid (tissue - could be real volume like cubic mm or a weigh like mg, sometimes that's called quantity instead of volume). 

Concentration, as long as the SI prefix is the same for both units it should be fine, e.g. GDC specifies milligram/milliliter which would be the same numerical values as micrograms/microliter. The problem is when you start seeing them mixed and ensuring people convert them correctly, I see ng/ul on a few of the manifests for example.

Reply:
Where else do we have units? Maybe we should fk units to another unit table to ensure we have a strict vocab?

Reply:
@dankolbman we have units in biospecimen only

Reply:
@allisonheath  Oh I'm sorry, I meant that I have seen ng/ul for most of the x01 cohorts I've looked at not ug/ul. Hmm I don't know if we need a separate table for units just yet. I think the only fields that require units right now are these two in biospecimen, concentration and volume. Will we eventually start tracking weight and height in phenotypes?

Reply:
Ok. Let's rename `concentration` to `concentarion_mg_per_ml` and volume to `volume_ml` and  make sure to convert values to these units while importing data.